src/partest/scala/tools/partest/ConsoleLog.scala

```diff
@@ -23,9 +23,13 @@ class ConsoleLog(colorEnabled: Boolean) {
   val magenta = colored(Console.MAGENTA)
 
   private[this] var dotCount = 0
-  private[this] val DotWidth = 72
+  private[this] final val DotWidth = 72
 
-  def leftFlush(): Unit = {
+  def print(text: String) = synchronized {
```

Comment:
So is this to shadow Predef's?

Reply:
Since it all drains through `print` and it's just test code, this seemed like the fast path to happiness while remaining within scope.